src/harness/reference_models/iap/test_data/pal_2.json

```diff
@@ -0,0 +1,24 @@
+{
+  "palId": "pal_2",
+  "licensee": "CBSD Operator No.1",
+  "userID": "user_2",
+  "registrationInformation": {
+    "frn": "test_fcc_frn_pal_2",
+    "callSign": "call_sign_pal_2",
+    "licenseConditions": "license_pal_2"
+  },
+  "license": {
+    "id": "test_fcc_id_pal_2"
+  },
+  "licenseStatus": "VALID",
+  "channelAssignment": {
+    "primaryAssignment": {
+      "lowFrequency": 3570000000.0,
+      "highFrequency": 3575000000.0
```

Comment:
PAL frequency assignments are 4 channels maximum (40 MHz max)

Reply:
Updated   to 10MHz